assets/README.md

```diff
@@ -45,6 +45,7 @@ commit.
 ```bash
 > go generate .
 > git add bindata.go
+> git add bindata_version_hash.go
```

Comment:
@lidel it looks like you forgot to run this while rebasing as I don't see any changes to the bindata_version_hash.go file.

Reply:
@Stebalien merged #7609 and I rebased on top and added the relevant files.

Reply:
Tested locally and looks good, super happy about deterministic assets :+1:

Reply:
absolutely, it also helped with the fact that different OSes would give different modes.